app/components/UserValidator/Validator.css

```diff
@@ -1,6 +1,7 @@
 @import '~app/styles/variables.css';
 
 .overlay {
+  z-index: 100;
```

Comment:
:shipit:

Reply:
This is required to display the overlay with feedback after scanning in front of the scanner modal. I agree that you could lower the number, but really does it make a difference? If you have an alternative suggestion to this approach, please let me know. ✌🏻